src/models/zero_model.py

```diff
@@ -0,0 +1,12 @@
+import torch
+from core import ContextModel
+
+class ZeroModel(ContextModel):
+    def __init__(self, **kwargs):
+        super(ZeroModel, self).__init__()
+        self.name = "zero_model"
+        self.context_length = -1
+
+    def forward(self, xs, ys):
+        
+        return 0*ys
```

Comment:
This won't always be the right return shape -- consider where `xs` is longer than `ys` by one in the sequence length dimension

Reply:
I will fix this

Reply:
Nevermind, saw you already fixed it